test/components/connect.spec.js

```diff
@@ -2121,7 +2120,7 @@ describe('React', () => {
 
       @connect(null)
       class Parent extends React.Component {
-        componentWillMount() {
+        UNSAFE_componentWillMount() {
```

Comment:
When I tried to change it to `componentDidMount` it broke the test. Maybe it should be changed to `constructor`, I'm not sure

Reply:
What error did you get with cDM? The cWM function is not consequential here anyways; [this is testing for unmount cleanups](https://github.com/reactjs/react-redux/commit/ea53d6fb076359a864a1d543568d951d4b3eab3d). So, I'd be fine with removing it and putting a dispatch outside of the ReactDOM.render.

Reply:
`data` was null

Reply:
I think I'll just change the initial state